src/Illuminate/Foundation/Console/stubs/test.database_migration.stub

```diff
@@ -0,0 +1,8 @@
+<?php
+
+class DummyClass extends \TestCase
+{
+    use \Illuminate\Foundation\Testing\DatabaseMigrations;
+    //
```

Comment:
// <-- I added those cause I noticed the Model class had these src/Illuminate/Foundation/Console/stubs/model.stub